dubbo-common/src/main/java/org/apache/dubbo/common/io/Bytes.java

```diff
@@ -620,6 +620,9 @@ public static String bytes2base64(final byte[] bs, final int off, final int len,
         if (len < 0) {
             throw new IndexOutOfBoundsException("base642bytes: length < 0, length is " + len);
         }
+        if (len == 0) {
```

Comment:
pls. consider to add the same check for *public static byte[] base642bytes(final String str, final int off, final int len, final char[] code) *